tests/filesTest.js

```diff
@@ -151,24 +154,24 @@ describe('Main: Currently testing files management,', function () {
                   'd:getlastmodified', '',
                   MatchersV3.date('EEE, d MMM yyyy HH:mm:ss z', 'Mon, 19 Oct 2020 03:50:00 GMT')
                 )
-                .appendElement('d:getcontentlength', '', MatchersV3.number(11))
                 .appendElement('d:resourcetype', '', resourceType)
                 .appendElement(
                   'd:getetag', '',
-                  MatchersV3.regex('"[a-z0-9]+"', '"3986cd55c130a4d50ff0904bf64aa27d"')
+                  MatchersV3.regex(
+                    '"[a-z0-9]+"',
+                    '&quot;3986cd55c130a4d50ff0904bf64aa27d&quot;'
```

Comment:
why do we need to use `&quot;` here and not simply `"`?

Reply:
was just a copy paste. I will update it